docs/tutorials/RemovingFields.adoc

```diff
@@ -0,0 +1,125 @@
+= Tutorial - Removing a field from an entity
+:site-section: DeveloperGuide
+:toc: macro
+:toc-title:
+:imagesDir: ../images/remove
+:stylesDir: ../stylesheets
+:xrefstyle: full
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:warning-caption: :warning:
+:source-highlighter: highlightjs
```

Comment:
e.g. this

Reply:
Yeah I'm going through it again with a fine-comb. Fixing a lot of old problems.